GUI/guiBrowser.xml

```diff
@@ -1171,7 +1171,38 @@
 				<attributes>
 					<int name="Id" value="10020" />
 					<string name="Caption" value="&quot;warmup&quot;" />
-					<rect name="Rect" value="169, 160, 319, 190" />
+					<rect name="Rect" value="169, 140, 319, 177" />
+					<position name="MinSize" value="1, 1" />
+					<position name="MaxSize" value="0, 0" />
+					<enum name="LeftAlign" value="lowerRight" />
+					<enum name="RightAlign" value="lowerRight" />
+					<enum name="TopAlign" value="lowerRight" />
+					<enum name="BottomAlign" value="lowerRight" />
+					<bool name="Visible" value="true" />
+					<bool name="Enabled" value="true" />
+					<bool name="TabStop" value="false" />
+					<bool name="TabGroup" value="false" />
+					<int name="TabOrder" value="-1" />
+					<bool name="NoClip" value="false" />
+					<bool name="Border" value="false" />
+					<bool name="OverrideColorEnabled" value="false" />
+					<bool name="OverrideBGColorEnabled" value="false" />
+					<bool name="WordWrap" value="true" />
+					<bool name="Background" value="false" />
+					<color name="OverrideColor" value="65ffffff" />
+					<color name="BGColor" value="fad2d2d2" />
+					<enum name="HTextAlign" value="lowerRight" />
+					<enum name="VTextAlign" value="lowerRight" />
+				</attributes>
+
+			</element>
+
+			<element type="staticText">
+
+				<attributes>
+					<int name="Id" value="10041" />
+					<string name="Caption" value="&quot;current players&quot;" />
+					<rect name="Rect" value="10, 380, 320, 220" />

```

Comment:
Need to look into it but this makes it not render on staging. Swapping 220 and 380 makes it show "correctly", although alignment is weird (didn't have time to check vanilla right now to see if it was just like this already). Might be better to align to the bottom of the window or something if that space is unused?

Also letting this define the alignment instead of hardcoding it in engine.